srcpkgs/wlroots0.17/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'wlroots0.17'
+pkgname=wlroots0.17
+version=0.17.0
+revision=1
+build_style=meson
+# Follow upstream packaging recommendations:
+# https://gitlab.freedesktop.org/wlroots/wlroots/-/wikis/Packaging-recommendations
+configure_args="--auto-features=enabled -Dexamples=false -Dwerror=false
+ -Db_ndebug=false"
+hostmakedepends="pkg-config wayland-devel glslang hwids"
+_devel_depends="libgbm-devel libglvnd-devel libseat-devel eudev-libudev-devel
+ libdrm-devel libinput-devel libxkbcommon-devel pixman-devel wayland-devel
+ wayland-protocols vulkan-loader Vulkan-Headers xcb-util-errors-devel
+ xcb-util-wm-devel xcb-util-renderutil-devel"
+makedepends="${_devel_depends}
+ libxcb-devel xcb-util-cursor-devel xcb-util-devel xcb-util-image-devel
+ xcb-util-keysyms-devel xcb-util-xrm-devel xorg-server-xwayland libdisplay-info-devel"
```

Comment:
Put `libdisplay-info-devel` in `_devel_depends` instead, so it doesn't have to be pulled in by every template that wants to depend on `wlroots0.17-devel`.